raiden-ts/src/raiden.ts

```diff
@@ -1474,20 +1474,30 @@ export class Raiden {
             mergeMap(async (response) => response.text()),
             map((text) => {
               const suggestions = decode(SuggestedPartners, jsonParse(text));
-              assert(suggestions.length, ['empty partner suggestion from PFS', { pfs: pfs.url }]);
+
+              if (!suggestions.length) {
+                firstResponse = [];
+                throw Error('PFS returned no suggested partners');
```

Comment:
Nitpick: prefer `throw new Error()`, since Error is a constructor.

Reply:
✅